plugins/mobile-plugin/src/test/java/pagefactory/mobile/pages/auth/EmailPage.java

```diff
@@ -0,0 +1,27 @@
+package pagefactory.mobile.pages.auth;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.mobile.MobilePage;
+import ru.sbtqa.tag.pagefactory.Page;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'ru.sbtqa.tag.pagefactory.Page'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)